src/test/ui/methods/assign-to-method.rs

```diff
@@ -1,24 +0,0 @@
-// compile-flags: -Zsave-analysis
```

Comment:
This test however existed before save-analysis and was repurposed to test a save-analysis ICE at the same time. We can just remove the compile flag and keep the test.